StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -505,7 +510,8 @@ extension PaymentSheet {
                 let shouldSave = false // always false, as we don't show a save-to-merchant checkbox in Link VC
 
                 if elementsSession.linkPassthroughModeEnabled {
-                    linkAccount.sharePaymentDetails(id: paymentDetails.stripeID, cvc: paymentDetails.cvc) { result in
+                    // allowRedisplay is nil since we are not saving a payment method.
+                    linkAccount.sharePaymentDetails(id: paymentDetails.stripeID, cvc: paymentDetails.cvc, allowRedisplay: nil) { result in
```

Comment:
@davidestes  - a bit unclear here on capturing allowRedisplay.  Is it possible to have a checkbox when given .withPaymentDetails?

Reply:
Chatted offline -- we are not showing a save payment method checkbox in the link flow, so we will never be saving the payment method back to the user, therefore there is no reason to send an allow_redisplay value.